### PR TITLE
Update Dockerfile and dependencies

### PR DESCRIPTION
**Pull request by ElfoLiNk**
Update base image to alpine:3.7
Update glibc version to 2.27-r0

### CODE REVIEW

File: deploy/build [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
```
Comment (dashpole) [inline deploy/build]: why is this file added?

Comment (ElfoLiNk) [reply deploy/build]: Wrongly added. Just removed
